src/helpers/tests/generator.test.ts

```diff
@@ -357,7 +381,9 @@ describe("saveFile", () => {
       generator.saveFile({ sourceFile, generatedFilePath: testFilePath });
     }).not.toThrow();
 
-    expect(fs.existsSync(testFilePath)).toBe(true);
+    expect(fs.existsSync(testFilePath)).toBe(true); // double check
+    fs.unlinkSync(testFilePath);
+    expect(fs.existsSync(testFilePath)).toBe(false);
```

Comment:
ahh we can remove this if were already doing it in the afterEach, my bad I missed that